cypress/page/menu-content.page.ts

```diff
@@ -4,15 +4,15 @@ class MenuContentPage {
 
   constructor() {
     this.menuContentPageURL = "http://automationpractice.com/";
-    this.tShirtMenu = "#block_top_menu > ul > li:nth-child(3) > a";
+    this.tShirtMenu = "T-shirts";
```

Comment:
instead of using contains in this one, could you try to use the title attribute of the `li` tag, instead of the `nth-child`, in that way you can use only one get and you don't have to concatenate so many commands